### PR TITLE
[devtools] export DevtoolsPluginClient

### PR DESCRIPTION
**Pull request by cyrilbo**
# Why

https://github.com/expo/dev-plugins/pull/27#discussion_r1489341913

# How

# Test Plan

# Checklist

- [x] Documentation is up to date to reflect these changes (eg: https://docs.expo.dev and README.md).
- [x] Conforms with the [Documentation Writing Style Guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md)
- [x] This diff will work correctly for `npx expo prebuild` & EAS Build (eg: updated a module plugin).

### CODE REVIEW

**Comment by Kudo** [comment]
thanks for helping the change. at expo, we ship and commit transpiled js files. could you help to run the following command and commit the changes?
```sh
$ cd /path/to/expo/expo/packages/expo
$ yarn expo-module prepare
```

**Comment by cyrilbo** [comment]
@Kudo Sorry I missed that, I've committed the changes in the build folder :)

File: packages/expo/CHANGELOG.md [hunk 1]
```diff
@@ -16,6 +16,7 @@
 - Mark the exp prop as optional, this is only used in Expo Go ([#27095](https://github.com/expo/expo/pull/27095) by [@brentvatne](https://github.com/brentvatne))
 - [expo-updates] Migrate to requireNativeModule/requireOptionalNativeModule. ([#25648](https://github.com/expo/expo/pull/25648) by [@wschurman](https://github.com/wschurman))
 - Remove implicit dependency on expo-updates to do runtime version check at runtime. ([#26080](https://github.com/expo/expo/pull/26080) by [@wschurman](https://github.com/wschurman))
+- Export DevToolsPluginClient from devtools. ([#27125](https://github.com/expo/expo/pull/27125) by [@cyrilbo](https://github.com/cyrilbo))
```
Comment (Kudo) [inline packages/expo/CHANGELOG.md:19]:
```suggestion
- Export `DevToolsPluginClient` from `expo/devtools`. ([#27125](https://github.com/expo/expo/pull/27125) by [@cyrilbo](https://github.com/cyrilbo))
```

**Review by Kudo — APPROVED** [review]
thanks for having the work. i'm going the merge this and plan for new expo package publish

**Pull request merged**